cookiecutter.json

```diff
@@ -13,6 +13,18 @@
     "use_flower": "n",
     "use_fingerprinting": "n",
     "use_channels": "y",
+    "use_allauth": "n",
+    "allauth_trust_external_emails": "y",
+    "use_allauth_apple": "n",
+    "use_allauth_atlassian": "n",
+    "use_allauth_discord": "n",
+    "use_allauth_facebook": "n",
+    "use_allauth_github": "n",
+    "use_allauth_gitlab": "n",
+    "use_allauth_google": "n",
+    "use_allauth_microsoft": "n",
+    "use_allauth_openid_connect": "n",
+    "use_allauth_twitter": "n",
```

Comment:
Nitpick (not a call-to-action):
I may be the only person who suggests what I gonna suggest, but I'd rather make it a list, cause we already have too many cookiecutter questions:
```
# cookiectutter
"allauth_providers": "apple,atlassian,google,...",

# other files
{% if 'google' in cookiecutter.allauth_providers %}
```